packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/defaultSearchBoxInput.html

```diff
@@ -37,6 +37,8 @@
               <template if:false={isQueryEmpty}>
                 <lightning-button-icon
                   onclick={clearInput}
+                  onkeyup={onKeyup}
+                  onfocus={onClearButtonFocus}
```

Comment:
this makes sure it adds the `is-focused` attribute on the clear icon for the check in the `handleEnter()` function